src/qibolab/dummy.py

```diff
@@ -60,6 +62,14 @@ def create_dummy(with_couplers: bool = True):
     channels["twpa"].local_oscillator = twpa_pump
 
     qubits, couplers, pairs = load_qubits(runcard)
+    if with_kernels:
+        kernels = Kernels.load(path=extras_folder / "kernels.npz")
+        for q in kernels.data.keys():
+            # To handle Kernel.save() using strings
+            if q in qubits.keys():
+                qubits[q].kernel = kernels[q]
+            else:
+                qubits[int(q)].kernel = kernels[q]
```

Comment:
For dummy I would actually avoid to do any of this manipulations and just put an array directly in `qubits[q].kernel`. I believe that this should be an issue for qibocal but I need to check.

Reply:
I don't think this involves qibocal as it is the initial loading of parameters on qibolab.

Reply:
Not needed thanks to the load and dump help you provided